t/TwitchFeatured.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+ddg_spice_test(
+    [
+        'DDG::Spice::Twitch::Featured'
+    ],
+    'twitch featured' => test_spice(
+        '/js/spice/twitch/featured/',
+        call_type => 'include',
+        caller => 'DDG::Spice::Twitch::Featured',
+        proxy_cache_valid => "418 1d"
+    ),
+    'twitchtv featured' => test_spice(
+        '/js/spice/twitch/featured/',
+        call_type => 'include',
+        caller => 'DDG::Spice::Twitch::Featured',
+        proxy_cache_valid => "418 1d"
+    ),
+    'featured twitch' => test_spice(
+        '/js/spice/twitch/featured/',
+        call_type => 'include',
+        caller => 'DDG::Spice::Twitch::Featured',
+        proxy_cache_valid => "418 1d"
+    ),
+    'featured twitchtv' => test_spice(
+        '/js/spice/twitch/featured/',
+        call_type => 'include',
+        caller => 'DDG::Spice::Twitch::Featured',
+        proxy_cache_valid => "418 1d"
+    ),
+    'twitch featured insert random stuff' => undef, 
+    'what is twitch featured' => undef, 
+    'about twitch' => undef,
```

Comment:
@Dwaligon Do you mind adding one more `undef` test here and pushing, just to force Travis CI to restart the testing process :+1: Once complete this will be right to go :smile: 

Sorry for any inconvenience the failed tests was caused by an internal change.